its/autoscan/src/test/resources/autoscan/diffs/diff_S6485.json

```diff
@@ -1,6 +1,6 @@
 {
   "ruleKey": "S6485",
   "hasTruePositives": false,
-  "falseNegatives": 0,
+  "falseNegatives": 8,
```

Comment:
Looks like many FN, do you know from where this is coming?

Reply:
S6485 only runs if:
```java
  @Override
  public boolean isCompatibleWithJavaVersion(JavaVersion version) {
    return version.isJava19Compatible();
  }
```
So previously, there was no issue at all with or without semantics. Now, we have FN without semantic.